### DISCUSSION TITLE
文字书写问题写错 false 写成 falsy

### DISCUSSION THREAD

**Discussion by wangyigui11** [vuejs/router#1913] [Help and Questions] [answered]
### Reproduction

None

### Steps to reproduce the bug

![router-error](https://github.com/vuejs/router/assets/59229339/a267f086-3ddc-4e4c-82f1-07a7a98e87eb)

### Expected behavior

期望 falsy  ==> false

### Actual behavior

文字错误

**Comment by edison1105** [comment, marked as answer]
这并不是一个拼写错误。
`falsy` 意味着可以是 `0`, `null`, `undefined` 等

**Comment by wangyigui11** [comment]
好的， 谢谢




王一
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "vuejs/router"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年7月3日(星期一) 下午4:41
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [vuejs/router] 文字书写问题写错 false 写成 falsy (Issue #1911)





 
这并不是一个拼写错误。
 falsy 意味着可以是 0, null, undefined 等
